dali/base/dadfs.cpp

```diff
@@ -1243,13 +1243,11 @@ static void setUserDescriptor(Linked<IUserDescriptor> &udesc,IUserDescriptor *us
     udesc.set(user);
 }
 
+static bool scopePermissionsAvail = true;
```

Comment:
This bool is now publicly accessible. Consider encapsulating it along with a getter and the getScopePermissions method.